src/components/number-picker/number-picker.component.ts

```diff
@@ -207,5 +211,10 @@ export class NumberPickerComponent implements ControlValueAccessor, OnDestroy, O
         this._propagateChange(value);
     }
 
+    /** Determine precision based on the length of passed number (number of digits) */
+    determinePrecision(value: number): number {
```

Comment:
By doing this this completely ignores the value the user may have provided as the precision input, it will always either be the default precision or large precision.

Is there an issue with just setting the default precision value to the large precision? That way the consumer is free to override it if they need?

Reply:
Yes for some reason that didn't work for me when I tested it the first time around. I've re-evaluated the solution.